compiler/testData/diagnostics/tests/regressions/kt282.fir.kt

```diff
@@ -14,6 +14,6 @@ fun f(): Unit {
     val i : Int? = null
     i <!NONE_APPLICABLE!>+<!> 1
     set + 1
-    1 <!INAPPLICABLE_CANDIDATE!>in<!> set
+    1 <!UNSAFE_CALL!>in<!> set
```

Comment:
For my own record, this and above should be `UNSAFE_OPERATOR_CALL`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/regressions/kt282.kt#L17

Reply:
Done at https://github.com/JetBrains/kotlin/pull/4074/files#diff-30a21c37fc436b587141fc7c83bd1e5a73d90efdeac354179605d7a984bda37b